dev_guide/builds/build_strategies.adoc

```diff
@@ -577,28 +577,37 @@ jenkinsPipelineStrategy:
 
 [NOTE]
 ====
-Specifics on the mapping between `BuildConfig` environment variables and Jenkins job parameters:
-
-* When a Jenkins job is created or updated based on changes to a Pipeline strategy `BuildConfig`, any environment
-variables in the `BuildConfig` are mapped to Jenkins job parameters definitions, where the default values for the Jenkins job parameters
-definitions are the current values of the associated environment variables.
-* You can still add additional parameters (whose names differ from the names of the environment variables in the `BuildConfig`)
-to the Jenkins job from the Jenkins console after the job's initial creation and they will be honored when builds are started for those Jenkins jobs.
-* How you start builds for the Jenkins job then dictates how the parameters are set.
-* If you start via `oc start-build`, the values of the environment variables in the `BuildConfig` are what the parameters are set
-to for the corresponding job instance.  Any changes you make to the parameters default values from the Jenkins console are ignored.
-The `BuildConfig` values take precedence.
-* If you start via `oc start-build -e`, the values for the environment variables specified in the `-e` option take precedence.  And if you
-specify an environment variable not listed in the `BuildConfig`, they will be added as a Jenkins job parameter definitions. And, again, any
-changes you make from the Jenkins console to the parameters corresponding to the environment variables are ignored.  The `BuildConfig`
-and what you specify via `oc start-build -e` takes precedence.
-* If you start the Jenkins job via the Jenkins console, then you can control the setting of the parameters via the Jenkins console as part
-of starting a build for the job.
-
-Please visit the xref:../../dev_guide/builds/build_environment.adoc#dev-guide-build-build-environment[build environment]
-section for more advanced usage.
-
 You can also manage environment variables defined in the `BuildConfig` with the
 xref:../../dev_guide/environment_variables.adoc#dev-guide-environment-variables[`oc set env`] command.
```

Comment:
this is already covered in this topic in general up on line 162 so i think this can be removed.  (it applies equally to all the build strategies).